storage/src/storage/ArrayStorage.java

```diff
@@ -0,0 +1,109 @@
+package storage;
+
+import model.Resume;
+import storage.exception.FullStorageException;
+import storage.exception.ResumeAlreadyStoredException;
+import storage.exception.ResumeNotFoundException;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public abstract class ArrayStorage implements Storage {
+    protected static final int NOT_EXISTING_INDEX = -1;
+    protected static final int DEFAULT_MAX_STORAGE_SIZE = 100000;
+    protected int maxSize;
+    protected int size;
+
+    protected Resume[] storage;
+
+    public ArrayStorage() {
```

Comment:
Look at SonarLint hints. It's a minor thing but there is actually no point in `public` modifier as you can't make any use of it. Should be `protected`

Reply:
Looks like you will have exceptions in case of using this constructor. Storage is not initialized

Reply:
Done